employees/serializers.py

```diff
@@ -49,8 +49,8 @@ class Meta:
             'date',
             'project',
             'author',
-            'description',
             'work_hours',
+            'description',
```

Comment:
Why this change?

Reply:
I changed it because  I wanted to show below description field that it is supported by Markdown. I didn't know how to resolve it better, as form is rendering by serializer so I changed it like that. Maybe there is better solution. :)

Reply:
You can add `helptext` to model field, check this:
https://github.com/Code-Poets/sheetstorm/pull/87/files#diff-dd359dbd34d8fff0ddd29ecb0703e366R33

and then it gets automatically displayed when rendering whole form. But this is optional, as I guess we are going to change all our templates soon right? Just FYI that it can be achieved without having to put it at the end.

Reply:
Ok so I will delete this, because as you said templates will be changed so I think it is not very necessary. Thanks!